lib/go-parser/index.ts

```diff
@@ -151,23 +151,32 @@ export async function goModulesToScannedProjects(
       await eventLoopSpinner.spin();
     }
 
-    const depGraph = await new GoBinary(goBinary).depGraph();
+    try {
+      const depGraph = await new GoBinary(goBinary).depGraph();
+      if (!depGraph) {
+        continue;
+      }
 
-    if (!depGraph) {
-      continue;
+      const depGraphFact: DepGraphFact = {
+        type: "depGraph",
+        data: depGraph,
+      };
+      scanResults.push({
+        facts: [depGraphFact],
+        identity: {
+          type: DEP_GRAPH_TYPE,
+          targetFile: filePath,
+        },
+      });
+    } catch (err) {
+      if (err instanceof MissingPCLNTableError) {
+        // Depending on how a CGo binary is built, it might not have the PCLN
+        // table linked into the binary. In such cases, we don't want to fail,
+        // we simply don't want to scan the binary.
+        continue;
+      }
+      throw err;
```

Comment:
is this being caught somewhere? I'm not sure it's a good idea to fail the entire scan in case of an error. I would prefer catching that error, print a debug message, and move on. IMO it's better to have _some_ scan results than break the entire flow because one binary had an unexpected error

Reply:
>  IMO it's better to have some scan results than break the entire flow because one binary had an unexpected error

Yeah I absolutely agree on that. I'll add a debug-message and remove the throw. I guess we also don't need the new class then.

I'll keep the linter adjustment though, that has been bothering me for a while now :D